src/index.html

```diff
@@ -17,11 +17,696 @@
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <a
+            href="#"
+            class="card__image"
+          >
+            <img
+              src="./images/imac.jpeg"
+              alt="APPLE A1419 iMac 27"
+            />
+          </a>
+
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__rating">
+            <div class="rating-stars">
+              <div class="stars stars--0 stars--hidden">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--1 stars--hidden">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--2 stars--hidden">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--3 stars--hidden">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--5 stars--hidden">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+            </div>
+            <span class="rating-reviews">Reviews: 5</span>
+          </div>
+          <p class="card__price">
+            <span class="price-text">Price:</span>
+            <span class="price-value">$2,199</span>
+          </p>
+          <button
+            class="card__buy-button"
+            data-qa="card-hover"
```

Comment:
The 'data-qa' attribute value 'card-hover' is used here, but on line 262, the value 'hover' is used. Consider using consistent naming for 'data-qa' attributes across similar elements.